src/nikhilMJun20/StringEquality.java

```diff
@@ -0,0 +1,38 @@
+/*
+Coding test 10 :
+
+Print minimum rotation required to compare both strings equality.
+
+Example 1 :
+String input1 = "abcdef";
+String input2 = "defabc";
+output : 3
+
+Example 2 :
+String input1 = "adbcdef"; -> push right
+fadbcde
+efadbcd
+defadbc
+String input2 = "defadbc";
+output : 3
+*/
+
+package nikhilMJun20;
+
+public class StringEquality {
+	int getRotationCount(String input1, String input2) {
+		int count = 0;
+		while(! input1.equals(input2)) {
```

Comment:
@NikhilMJun20 , For positive scenario it looks super good. 
Just a suggestion.
we should add one more condition in while loop to handle negative scenario as well otherwise if two string could not compare at all then it would be infinite loop.

while(!input1.equals(input2) && count <= input1.length)

Reply:
@KrishnaKTechnocredits 👍 
Yes, I missed this condition.
I will keep in mind...